docs/api/hl.rst

```diff
@@ -0,0 +1,12 @@
+.. _hl_api:
+
+High Level API
+==============
+
+.. automethod:: jupyter_kernel_mgmt.hl.start_kernel_async
```

Comment:
Nitpick: I'd like to document these without referring to the `hl` module, e.g.  `jupyter_kernel_mgmt.start_kernel_async`. That's effectively an implementation detail.

Reply:
I have tried to define a currentmodule like in the othe rst files but I receive for the methods the following warning and at the end, the methods are not generated.

WARNING: don't know which module to import for autodocumenting 'start_kernel_async' (try placing a "module" or "currentmodule" directive in the document, or giving an explicit module name)

Reply:
Tried with other configuration, but not sure how sphynx handles the top level methods instead of the class methods.